tiktokpy/models/feed.py

```diff
@@ -80,8 +80,8 @@ class VideoInfo(BaseModel):
     duration: int
     ratio: str
     cover: HttpUrl
-    play_addr: HttpUrl
-    download_addr: HttpUrl
+    play_addr: Optional[HttpUrl]
```

Comment:
= None?

Reply:
So I assume you would keep this line as it is, right?